elasticsearch/src/test/scala/docs/scaladsl/ElasticsearchConnectorBehaviour.scala

```diff
@@ -378,7 +378,7 @@ trait ElasticsearchConnectorBehaviour {
 
         val indexName = "sink7"
         val createBooks = Source(books)
-          .map { book: (String, Book) =>
+          .map { (book: (String, Book)) =>
```

Comment:
Fixed syntax in Elasticsearch, but more intricate things made me not enable Scala 3, yet.